src/plugins/quickdump/README.md

```diff
@@ -0,0 +1,114 @@
+- infos = Information about the quickdump plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/quickdump
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = tested nodep libc compatible preview
+- infos/metadata =
+- infos/description = much quicker version of dump
+
+## Introduction
+
+`quickdump` is a storage plugin based on the `dump` format. It is a lot quicker than the old `dump` plugin, because it does not use commands
+and stores string lengths as binary data. Through these changes all string comparisons and integer-string conversions can be eliminated,
+which made up for a lot of the time spent by the `dump` plugin.
+
+The format is also useful for IPC and streaming, because of this it is used by the `specload` plugin.
```

Comment:
and pluginprocess?

Reply:
As soon as it is implemented (new PR so this can be merged quicker), I will add a note.